src/test/java/com/problem1/myproject/MyprojectApplicationTests.java

```diff
@@ -2,12 +2,18 @@
 
 import org.junit.jupiter.api.Test;
 import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.context.annotation.Bean;
+import org.springframework.security.crypto.bcrypt.BCryptPasswordEncoder;
+import org.springframework.security.crypto.password.PasswordEncoder;
 
 @SpringBootTest
 class MyprojectApplicationTests {
 
 	@Test
 	void contextLoads() {
 	}
-
+	@Bean
+	PasswordEncoder passwordEncoder(){
```

Comment:
@RobertRodeapps why did you define this bean in a test class?

Reply:
I defined it when i have not understood very well what @Mock Does and i forgot about it, i will erase it